docker-compose.dev.yml

```diff
@@ -28,9 +28,18 @@ services:
             - ./server/modules:/server/modules
         environment:
             - WATCHFILES_FORCE_POLLING=true
+    prometheus:
+        image: prom/prometheus:latest
+        restart: always
+        volumes:
+            - ./prometheus.yml:/etc/prometheus/prometheus.yml
+        command:
+            - --config.file=/etc/prometheus/prometheus.yml
+        ports:
+            - 9090:9090
```

Comment:
put this behind nginx, at `/prometheus`, we can do this in the nginx.dev.conf like
```nginx
       location /prometheus/ {
            resolver 127.0.0.11 valid=15s;

            proxy_set_header   Host $host;
            set $upstream http://prometheus:9090;
            proxy_pass $upstream;

            rewrite ^/prometheus/?$ /prometheus/graph redirect;
            rewrite ^/prometheus(.*)$ $1 break;
        }
```

Reply:
```
rewrite ^/prometheus/?$ /prometheus/graph redirect;
```
I tried adding this line but it routes me to port 80 instead of port 3001

```
rewrite ^/prometheus/?$ http://$host:3001/prometheus/graph redirect;
```
this worked as `localhost:3001/prometheus/` would go to `localhost:3001/prometheus/graph` as intended but `localhost:3001/prometheus` goes to `localhost/prometheus/graph` again

should i still add this?

Reply:
resolved